hack/make/.ensure-syscall-test

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+# Build a C binary for cloning a userns for seccomp tests
+# and compile it for target daemon
+
+dir="$DEST/syscall-test"
+mkdir -p "$dir"
+(
+	if [ "$(go env GOOS)" = "linux" ]; then
+		cd "$dir"
+		gcc -g -Wall -static ../../../../contrib/syscall-test/userns.c -o ./userns-test
+		gcc -g -Wall -static ../../../../contrib/syscall-test/ns.c -o ./ns-test
+		gcc -g -Wall -static ../../../../contrib/syscall-test/acct.c -o ./acct-test
```

Comment:
Do you think it's worth installing `gcc` in the container so we can compile and run these tests properly on a host whose arch doesn't match the one we're using? :smile: